middle_end/closure/closure.ml

```diff
@@ -117,10 +118,15 @@ let prim_size prim args =
   match prim with
   | Pread_symbol _ -> 1
   | Pmakeblock _ -> 5 + List.length args
-  | Pfield _ -> 1
+  | Pfield(_, isptr, Mutable) ->
+      begin match isptr with
+      | Pointer -> 2
```

Comment:
Is this correct?

Reply:
The new code seems to make provisions for a barrier when reading from a mutable field. AFAIK, in the current Multicore OCaml design, reads from mutable fields are compiled like reads from immutable fields, so this change should probably be reverted.

Reply:
Reverted. Thanks for the clarification.